docs/user-guide.md

```diff
@@ -92,48 +93,106 @@ To grab the latest Akri Helm charts, run `helm repo update`.
+
+Let's walk through building an Akri installation command:
 
-1. When installing the Akri Helm chart, you can specify what Configuration to apply by specifying the protocol
-   that will be used in the Configuration. This is done in the setting `--set <protocol>.enabled=true` below. Here,
-   `<protocol>` could be `udev`, `onvif`, or `opcua`. Helm will automatically apply the default Configuration for that protocol to
-   the cluster. You can set values in the Helm install command to customize the Configuration. To explore the values you
-   can set, see our documentation on customizing the provided [ONVIF](./onvif-configuration.md),
-   [udev](./udev-configuration.md), and [OPC UA](./opcua-configuration.md) Configuration templates.
-
-    The Helm settings can also be used to customize where the Akri Controller runs. By default the Controller can be
-    deployed to any control plane or worker node. These settings can be changed by adding extra settings when installing
-    Akri below. If you don't want the Controller to ever be scheduled to control plane nodes, add `--set
-    controller.allowOnControlPlane=false` to your install command below. Conversely, if you only want the Controller to
-    run on control plane nodes, add `--set controller.onlyOnControlPlane=true`. This will guarantee the Controller only
-    runs on nodes with the label (key, value) of (`node-role.kubernetes.io/master`, ""), which is the default label for
-    the control plane node for Kubernetes.
-    
-    However, control plane nodes on MicroK8s and K3s may not have this exact label by
-    default, so you can add it by running `kubectl label node ${HOSTNAME,,} node-role.kubernetes.io/master=
-    --overwrite=true`. Or alternatively, in K3s, you can keep the default label value on the master and add `--set controller.nodeSelectors."node-role\.kubernetes\.io/master"=true` to the install command below.
-
-    Run the following to fetch the Akri Helm chart, install Akri, and apply the default configuration for `<protocol>`,
-    optionally specifying the image for the broker pod that should be deployed to utilize each discovered device.
+1. Get Akri's Helm repo
     ```sh
     helm repo add akri-helm-charts https://deislabs.github.io/akri/
-    helm install akri akri-helm-charts/akri \
-        $AKRI_HELM_CRICTL_CONFIGURATION \
-        --set <protocol>.enabled=true \
-        # --set <protocol>.brokerPod.image.repository=<your broker image> \
-        # apply any additional settings here
     ```
-    > Note: set `<protocol>.brokerPod.image.tag` to specify an image tag (defaults to `latest`).
+2. Install Akri's Controller and Agent, specifying the crictl configuration from [prerequisites above](#setting-up-your-cluster) in not using vanilla Kubernetes:
+    ```sh
+     helm install akri akri-helm-charts/akri \
```

Comment:
Should all the docs use akri-dev until our next release since they use the latest helm charts and discovery handler model?

Reply:
i don't know the best answer here.  it is kind of painful regardless.  could do this sort of thing: 'starting in the release after X.Y.Z, you can use this..."

Reply:
I feel like it is easier to just point to the dev chart and then switch back when we release.